faq/iam.mdx

```diff
@@ -65,29 +65,57 @@ For each group, a corresponding policy has been created, defining rules that giv
 
 <Concept>
 
-## What are applications and why do I need them?
+## What are applications, and why do I need them?
 Applications represent the identity of non-human users (such as a CI pipeline, a custom script, or a Terraform provider). They are used to attach permissions and API keys to these operations - without being linked to specific users and their own rights.
 
 </Concept>
 
 <Concept>
 
 ## Why do I have to select a preferred Project for Object Storage when I create an API key?
-Due to limitations on the Object Storage API, API keys cannot perform Object Storage actions on several projects at the same time. Whenever you generate an API key that will be used on Object Storage, you must specify a preferred project where the API key will be able to perform actions. For more information refer to the [Using IAM API keys with Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) documentation page. 
+Due to limitations on the Object Storage API, API keys cannot perform Object Storage actions on several projects at the same time. Whenever you generate an API key that will be used on Object Storage, you must specify a preferred project where the API key will be able to perform actions. For more information, refer to the [Using IAM API keys with Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) documentation page.
 
 </Concept>
 
 <Concept>
 
-## How can I set up a permission set more fine grained than ReadOnly or FullAccess (e.g. WriteOnly)? 
-Currently only FullAccess and ReadOnly permission sets are available for most Scaleway resources. More permission sets that allow finer-grained actions will be released in the upcoming months. 
+## Why and how should I rotate API keys?
+API keys are credentials that grant access to resources in Scaleway Organizations. Like any credential, it is good practice to change them on a regular basis to reduce the risk of security breaches.
+To change your API key, you can generate a new one for your user and/or application and delete the old API key.
 
 </Concept>
 
 <Concept>
 
-## Why and how should I rotate API keys?
-API keys are credentials that grant access to resources in Scaleway Organizations. Like any credential, it is good practice to change them on a regular basis to reduce the risk of security breaches.
-To change your API key, you can generate a new one for your user and/or application and delete the old API key.
+## How many API keys can I create?
+Each Organization can have up to 100 API keys.
```

Comment:
I wonder if this content would be better in a Limitations page or similar? See https://www.scaleway.com/en/docs/serverless/messaging/reference-content/limitations/ for example. Maybe we could discuss if we should have an overarching rule about where/how we give this kind of info?